src/main/java/seedu/heymatez/model/person/Email.java

```diff
@@ -12,21 +12,37 @@
 
     private static final String SPECIAL_CHARACTERS = "!#$%&'*+/=?`{|}~^.-";
     public static final String MESSAGE_CONSTRAINTS = "Emails should be of the format local-part@domain "
-            + "and adhere to the following constraints:\n"
-            + "1. The local-part should only contain alphanumeric characters and these special characters, excluding "
-            + "the parentheses, (" + SPECIAL_CHARACTERS + ") .\n"
-            + "2. This is followed by a '@' and then a domain name. "
+            + "and adhere to the following constraints:\n\n"
+
+            + "1. The entire email address should have a maximum length of 254 characters.\n\n"
+
+            + "2. The local-part should: \n"
+            + "    - be at least 2 character long \n"
+            + "    - only contain alphanumeric characters and these special characters, excluding "
+            + "the parentheses, (" + SPECIAL_CHARACTERS + ").\n\n"
+
+            + "3. This is followed by a '@' and then a domain name. "
             + "The domain name must:\n"
-            + "    - be at least 2 characters long\n"
+            + "    - be at least 2 characters long \n"
             + "    - start and end with alphanumeric characters\n"
-            + "    - consist of alphanumeric characters, a period or a hyphen for the characters in between, if any.";
+            + "    - consist of alphanumeric characters, a period or a hyphen for the characters in between, "
+            + "if any.\n\n"
```

Comment:
i think it would be better if it the space at the end of line 28 can be brought to line 29 just before "if" so that it is more consistent

Reply:
okay